content/departments/cloud/technical-docs/v2.0/onprem_data_migration.md

```diff
@@ -27,83 +27,7 @@ An operator must:
-    }
-  ]
-}
-```
-
-### Create snapshot contents
-
-#### Databases
-
-The customer should first be asked to create `pg_dump` exports of their Sourcegraph databases.
-`pg_dump` is designed to be [usable while the database is in use](https://www.postgresql.org/docs/current/app-pgdump.html):
-
-> It makes consistent backups even if the database is being used concurrently. pg_dump does not block other users accessing the database (readers or writers).
-
-Note that [we ask the customer to configure a notice](#prepare-instance) to let their users know that any actions taken after the point of the dump will not remain consistent on their new Cloud instance.
-
-Template commands for running `pg_dump` can be generated with `src snapshot databases` for various configurations:
-
-```sh
-$ src snapshot databases --help
-'src snapshot databases' generates commands to export Sourcegraph database dumps.
-Note that these commands are intended for use as reference - you may need to adjust the commands for your deployment.
-
-USAGE
-        src [-v] snapshot databases <pg_dump|docker|kubectl> [--targets=<docker|k8s|"targets.yaml">]
-
-TARGETS FILES
-        Predefined targets are available based on default Sourcegraph configurations ('docker', 'k8s').
-        Custom targets configuration can be provided in YAML format with '--targets=target.yaml', e.g.
-
-                primary:
-                        target: ...   # the DSN of the database deployment, e.g. in docker, the name of the database container
-                        dbname: ...   # name of database
-                        username: ... # username for database access
-                        password: ... # password for database access - only include password if it is non-sensitive
-                codeintel:
-                        # same as above
-                codeinsights:
-                        # same as above
-
-        See the pgdump.Targets type for more details.
-```
-
-Each of the generated commands must be run to completion to generate a database dump for each database.
-The output is as follows:
-
-- `src-snapshot/primary.sql`
-- `src-snapshot/codeintel.sql`
-- `src-snapshot/codeinsights.sql`
-
-For custom or complex database setups, the operator will decide how best to proceed, in collaboration with IE/CSE/etc - the goal in the end is to generate the above database dumps in a format aligned with the output of `src snapshot databases pg_dump` (the plain `pg_dump` commands).
-
-> NOTE: Database export may affect the performance of the Sourcegraph instance while it is in progress. Depending on the size of the instance's databases, this can take a very long time.
-
-#### Instance summary
-
-A snapshot summary is used to run acceptance tests post-migration. The customer should create one with `src snapshot summary` - note that a site admin access token is required:
-
-```sh
-src login # configure credentials for the instance
-src snapshot summary
```

Comment:
this command seems to be broken in practice
```
$ src snapshot summary
generate snapshot: error: 429 Too Many Requests
```

Reply:
What did you test this on? I didn't know Sourcegraph instances had built-in rate limiting at all 🤔

Reply:
I've tried it on a handful of sg instances and I haven't seen it work in practice yet...